addons/mail/models/mail_thread.py

```diff
@@ -276,10 +276,15 @@ def create(self, vals_list):
             threads_no_subtype = self.env[self._name]
             for thread in threads:
                 subtype = thread._creation_subtype()
-                if subtype:  # if we have a subtype, post message to notify users from _message_auto_subscribe
-                    thread.sudo().message_post(subtype_id=subtype.id, author_id=self.env.user.partner_id.id)
-                else:
+                if not subtype:
                     threads_no_subtype += thread
+                    continue
+                # if we have a subtype, post message to notify users from _message_auto_subscribe
+                thread.sudo().message_post(
+                    subtype_id=subtype.id, author_id=self.env.user.partner_id.id,
+                    # summary="o_mail_notification" is used to hide the message body in the front-end
+                    body=Markup(f'<div summary="o_mail_notification"><p>{thread._creation_subtype_message()}</p></div>')
```

Comment:
Can't we consider '_creation_subtype_message' (or equivalent if we modify a bit methods architecture) should return only text and escape it ? (aka: put it outside of markup). That way we are sure people won't do stupid things with it.